src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3442,11 +3420,7 @@ private ConversionOperatorDeclarationSyntax TryParseConversionOperatorDeclaratio
                     explicitInterfaceOpt = null;
                     opKeyword = this.EatToken(SyntaxKind.OperatorKeyword);
                     type = this.AddError(this.CreateMissingIdentifierName(), ErrorCode.ERR_IdentifierExpected);
-                    SyntaxToken open = SyntaxFactory.MissingToken(SyntaxKind.OpenParenToken);
-                    SyntaxToken close = SyntaxFactory.MissingToken(SyntaxKind.CloseParenToken);
                     var parameters = _pool.AllocateSeparated<ParameterSyntax>();
```

Comment:
this is making a builder just for an empty list.  can likely still simplify this further.